Quantum-Control-Applications-QuAM/Superconducting/quam_libs/experiments/two_qubit_xeb/README.md

```diff
@@ -0,0 +1,205 @@
+The user should also have access to the QuAM library, which can be found [here](https://docs.quantum-machines.co/quam/).
+We encourage you to reach out to the Customer Success team to set your QuAM accordingly. Moreover, note that this example might require dedicated adjustments depending on the way the gates are implemented on your platform.
+
+Prior to running the XEB example file `2q_xeb.py`, the user has to run the calibrations that define the gate and measurement parameters:
+- Single Qubit Gate: Implement a single qubit gate that will be used as a baseline for the random circuits. This gate should be calibrated to produce a $\pi$/2 rotation around the X axis (or equivalently the $SX$ gate as indicated [here](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.SXGate). On top of this that we will later refer to as the baseline gate, the user can also calibrate the other single qubit gates that will be used in the random circuits (e.g. $SY$, $SW$ or $T$ gates as done in [2] and [1]).
+- a Two-Qubit gate: Implement the two-qubit gate of interest, that will form the entangling layer in the experiment (together with the single qubit gates). For this flux-tunable use case, we will consider the $CZ$ (controlled phase) gate.
+- Calibrated Measurement Protocol for Qubit State Discrimination: Simultaneously measure the two-qubit system in its computational basis states ∣00⟩, ∣01⟩, ∣10⟩, ∣11⟩. Each qubit should have in its operations a readout operation called `"readout"`.
+
+Additionally, the user should install Qiskit to run the experiment. The installation instructions can be found [here](https://docs.quantum.ibm.com/start/install). 
+We offer the user the possibility to also leverage Qiskit Aer to simulate the XEB experiment on a noisy backend. To enable such simulation the user can also install Qiskit Aer by following the instructions [here](https://qiskit.github.io/qiskit-aer/getting_started.html).
+
+## How does the script work?
+
+The protocol relies on the OPX's ability to generate and execute in real-time random circuits on the quantum computer and compare the results with the ideal simulation. 
+The advantage of using QUA is that the randomization of the circuits can be done in parallel to circuit execution through real-time processing and random sampling.
+
+The user can choose which gate set to use to generate random unitaries. Usually, the experiment is performed with layers of random single-qubit gates followed by one fixed two-qubit gate, as depicted in the circuit below with the gate set formed by the gates $SX$, $SY$, and $SW$ [2]:
+[![XEB Circuit](images/xeb_circuit.png)](images/xeb_circuit.png)
+
+In this circuit, we have used the $SW$ gate defined in [2] as: 
+$$
+W^{1 / 2} \equiv R_{X+Y}(\pi / 2)=\frac{1}{\sqrt{2}}\left[\begin{array}{cc}
+1 & -\sqrt{i} \\
+\sqrt{-i} & 1
+\end{array}\right]
+$$
```

Comment:
This doesn't look like it renders properly

Reply:
I removed the full matrix, just left the logical effect and will leave the user to check the reference directly if necessary.